scm-ui/ui-components/src/repos/RepositoryEntry.tsx

```diff
@@ -131,10 +137,16 @@ class RepositoryEntry extends React.Component<Props> {
   };
 
   createTitle = () => {
-    const { repository } = this.props;
+    const { repository, t } = this.props;
+    const archivedFlag = repository.archived && (
+      <>
+        {" "}
+        <Icon name={"archive"} /> <Smaller>{t("repository.archived")}</Smaller>
+      </>
+    );
     return (
       <>
-        <ExtensionPoint name="repository.card.beforeTitle" props={{ repository }} /> <strong>{repository.name}</strong>
+        <ExtensionPoint name="repository.card.beforeTitle" props={{ repository }} /> <strong>{repository.name}</strong> {archivedFlag}
```

Comment:
We could fix the CSS for the star icon which is provided by the Landingpage plugin. Just need to check in the plugin if the repository is archived and adjust the color.